pkgs/tools/typesetting/rubber/default.nix

```diff
@@ -1,23 +1,34 @@
 { fetchurl, stdenv, python2Packages, texinfo }:
 
-stdenv.mkDerivation rec {
-  name = "rubber-1.3";
+let
+  pypkgs = python2Packages;
```

Comment:
That would be a simple search & replace. In any case, you can still use the let expression but don't use another name; everywhere `pythonPackages` is used so stick with it.